debezium-ddl-parser/src/main/antlr4/io/debezium/ddl/parser/mysql/generated/MySqlLexer.g4

```diff
@@ -1220,6 +1220,11 @@ STRING_USER_NAME:                    (
                                      (
                                        SQUOTA_STRING | DQUOTA_STRING 
                                        | BQUOTA_STRING | ID_LITERAL
+                                       | IP_ADDRESS
+                                     );
+IP_ADDRESS:                          (
+                                       [0-9]+ '.' [0-9.]+
+                                       | [0-9A-F:]+ ':' [0-9A-F:]+
```

Comment:
Should we add IPv6 address pattern as well?

Reply:
I think that's this very line, no?

@jpechane, I'm confused though by th ":" within the first character class, is this intended? And wouldn't the second one match a trainling ":", too? Worth adding a test for sure.

Reply:
Ah, and I guess this is not tolerant towards upper/lower-caseing?

Reply:
I.e. I'd roughly have expected something like this: `[0-9A-Fa-f] (':' [0-9A-Fa-f])*`.